### ISSUE TITLE
Bug: Namespaced properties won't correctly pass

### ISSUE THREAD

**Issue by petewalker** [openactive/data-model-validator#123]
The following documents are all equivalent:

```json
{
  "type": "Event",
  "name": "Event Name",
  "meetingPoint": "Event Meeting Point"
}
```

```json
{
  "@type": "Event",
  "schema:name": "Event Name",
  "oa:meetingPoint": "Event Meeting Point"
}
```

```json
{
  "@type": "Event",
  "http://schema.org/name": "Event Name",
  "https://www.openactive.org/ns#meetingPoint": "Event Meeting Point"
}
```

However, the validator will only pass the first one. All 3 scenarios (and combinations of) should also pass.

**Comment by petewalker** [comment]
@ldodds - just want to check my statement above is correct?

**Comment by petewalker** [comment]
Fixed in 93bc408
